pkg/cincinnati/cincinnati_test.go

```diff
@@ -10,11 +10,11 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/openshift/cluster-version-operator/pkg/clusterconditions/standard"
```

Comment:
Can we come up with some better names here? Both `clusterconditions` and `clusterconditions/standard` have a `ConditionRegistry` type and `NewConditionRegistry` constructor, and at callsites they will look like `clusterconditions.NewConditionRegistry` and `standard.NewConditionRegistry` and it is not clear how they are different and when to use which.